lib/templates.js

```diff
@@ -193,7 +193,7 @@ const createEventResourceTemplate = (serviceName, eventDefinition, appConfig, pa
         releaseStatus: "active",
         partOfPackage: _getPackageID(appConfig.ordNamespace, packageIds, 'event'),
         partOfGroups: [_getGroupID(serviceName, defaults.groupTypeId, appConfig)],
-        visibility: "public",
+        visibility: "public", // TODO: visibility is always set to public
```

Comment:
@Fannon @swennemers @zongqichen : As of now, the visibility of a resource is always set to `public`. 
I guess this should be changed to consider the (if existing) value of the `visibility` property in the `@ORD.Extensions`, like in the following example: 
`annotate ProcessorService with @ORD.Extensions: {
    title           : 'This is Processor Service title',
    shortDescription: 'short description for Processor Service',
    **visibility      : 'public',**
    extensible      : {supported: 'no'}
};`
Only then it would make sense to apply the filtering.
Am I correct?

Reply:
@aramovic79 : yes, via `ORD.Extensions: { visibility: 'internal' }` you can already set it to internal. Once you do that, filtering would apply.